discord/ext/commands/help.py

```diff
@@ -5,7 +5,7 @@
 Copyright (c) 2021-present Pycord Development
 
 Permission is hereby granted, free of charge, to any person obtaining a
-copy of this software and associated documentation files (the "Software"),
+copy of this software afnd associated documentation files (the "Software"),
```

Comment:
```suggestion
copy of this software and associated documentation files (the "Software"),
```

Looks like you accidentally introduced a spelling error here.